index.html

```diff
@@ -14,31 +14,36 @@ <h1>Hi, I'm Julia Świerczyńska</h1>
         <h2>I help your webs develop</h2>
     </header>
     <main>
-        <h2>About Me</h2>
+        <h2>Something about Me</h2>
         <section>
 
             <p>
-                Lorem ipsum dolor sit amet consectetur adipisicing elit. Pariatur adipisci nemo quidem commodi eveniet.
-                Voluptatem ipsum quis sunt. Similique eos consequatur perspiciatis exercitationem delectus in cum aut
-                corporis, nulla quidem.
-                Distinctio omnis, illo rem laudantium veritatis nesciunt repellat. Doloremque pariatur nulla doloribus
-                explicabo vero quod! Quos ducimus voluptatum molestiae fuga in nulla, perferendis mollitia quae et
-                nostrum, numquam voluptates deserunt?
-                Alias consequuntur incidunt officia dignissimos itaque debitis unde nobis nemo amet repudiandae,
-                delectus tempore non temporibus hic ipsam eaque velit? Facilis ipsum sit ducimus nesciunt aspernatur,
-                quaerat quod sequi repellat.
-                Sunt blanditiis expedita fugit possimus magnam odio voluptatum illo incidunt maiores numquam, ducimus
-                earum sit eos! Animi distinctio veritatis nostrum, fuga consequuntur deserunt cum soluta laborum neque
-                odio, est adipisci.
-                Adipisci ipsam iste, necessitatibus odio voluptates illum, nam ipsum autem ab dignissimos, minus
-                excepturi perferendis. Veniam ab sequi qui iure aut a illo at dolorem assumenda minima, culpa,
-                aspernatur perspiciatis?
+                I'm a Web Developer living in Poland. I enjoy creating intuitive and attractive webpages,
+                which will enhance the user experience. Sometimes, if you ask nicely, I can even take care of
+                copywriting.
+
+                Well-organised, independent and a quick-learner with a great sense of humour - and we're still speaking
+                of one person. My passions include psychology, martial arts, comic books and writing stories.
+                If I could be an animal, you would find me waddling with Gentoo penguins.
+
+                As an adventurous person, I'm not afraid of working on ambitious projects with a whole spectrum of
+                people.
+
+                Let's develop your pages!
+
```

Comment:
Pamiętaj, że HTML ignoruje wszystkie białe znaki poza pojedynczą spacją. Można to zrobić w takie sposób, ale wtedy zamiast elementu <p> trzeba użyć elementu <pre>. Natomiast najlepszą opcją byłoby po prostu podzielenie tego na osobne paragrafy (czyli <p>)